2023-12-28 16:00:40,757 INFO    StreamThr :8732 [internal.py:wandb_internal():89] W&B internal server running at pid: 8732, started at: 2023-12-28 16:00:40.757210
2023-12-28 16:00:40,757 DEBUG   HandlerThread:8732 [handler.py:handle_request():146] handle_request: status
2023-12-28 16:00:40,757 INFO    WriterThread:8732 [datastore.py:open_for_write():85] open: D:\paper\project\CENet-main\wandb\run-20231228_160040-5582fjsv\run-5582fjsv.wandb
2023-12-28 16:00:40,772 DEBUG   SenderThread:8732 [sender.py:send():382] send: header
2023-12-28 16:00:40,772 DEBUG   SenderThread:8732 [sender.py:send():382] send: run
2023-12-28 16:00:41,759 INFO    WriterThread:8732 [datastore.py:close():294] close: D:\paper\project\CENet-main\wandb\run-20231228_160040-5582fjsv\run-5582fjsv.wandb
2023-12-28 16:00:41,760 INFO    HandlerThread:8732 [handler.py:finish():866] shutting down handler
2023-12-28 16:00:41,774 ERROR   StreamThr :8732 [internal.py:wandb_internal():174] Thread SenderThread:
Traceback (most recent call last):
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\internal\internal_util.py", line 49, in run
    self._run()
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\internal\internal_util.py", line 100, in _run
    self._process(record)
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\internal\internal.py", line 328, in _process
    self._sm.send(record)
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\internal\sender.py", line 389, in send
    send_handler(record)
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\internal\sender.py", line 947, in send_run
    self._config_save(config_value_dict)
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\internal\sender.py", line 905, in _config_save
    config_util.save_config_file_from_dict(config_path, config_value_dict)
  File "D:\software\Anaconda\lib\site-packages\wandb\sdk\lib\config_util.py", line 87, in save_config_file_from_dict
    sort_keys=False,
  File "D:\software\Anaconda\lib\site-packages\yaml\__init__.py", line 200, in dump
    return dump_all([data], stream, Dumper=Dumper, **kwds)
TypeError: dump_all() got an unexpected keyword argument 'sort_keys'
